js/apps/events-page/dataServices/eventsService.js

```diff
@@ -49,8 +49,10 @@
 			return $q((resolve, reject) => {
 				$http.get(CONSTANTS.baseUrl + CONSTANTS.serviceUrls.events + '/' + id)
 					.then((response) => {
-						if (response.data && response.data.Description) {
-							response.data.Description = response.data.Description.replace(/<[\w/]+>/g, '');
+						if (response.data) {
+							if (response.data.Description) {
+								response.data.Description = response.data.Description.replace(/<[\w/]+>/g, '');
+							}
 							resolve(response.data);
```

Comment:
Do resolve and reject work inside a `then()`? I haven't seen that before.

Reply:
yes